### PR TITLE
Update README files

### LINKED ISSUE #76: Update documentation
The ReadMe files in all monitors are out of data with the change to the GSI-monitor repository and need to be updated.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Update README files in GSI-monitor to reflect current package organization and functionality.

Resolves #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 GSI Monitoring Tools
 
 These tools monitor the Gridpoint Statsical Interpolation (GSI) package's data assimiliation, detecting
```
Comment (ADCollard) [inline README.md]: "Statistical"

**Comment by EdwardSafford-NOAA** [comment]
Add examples of command line specifications, especially for -t|--tank.

File: src/Radiance_Monitor/README [hunk 2]
```diff
@@ -1,65 +1,75 @@
...
+$TANKDIR.
+
+5.  Run the image generation.  Navigate to GSI-monitor/src/Radiance_Monitor/image_gen/ush
+and run:
+
+ ./RadMon_IG_glb.sh suffix -p|--pdate -r|--run -n|--ncyc -t|--tank
+
+	suffix       $NET value or the name of your parallel.  
+        -p|--pdate   Cycle time for which you wish to generate images.
+                        It must be in YYYYMMDDHH format.
```
Comment (ADCollard) [inline src/Radiance_Monitor/README]: Can you specify that this is the FINAL cycle for which images will be generated (i.e, it is this cycle and `ncyc` cycles before this one).

File: src/Radiance_Monitor/README [hunk 3]
```diff
@@ -1,65 +1,75 @@
...
+                        specified the default value of 120 cycles is used.
+        -t|--tank    Location of the extracted data files.  This is likely to be your 
+                        comrot/$PSLOT directory.  This is only needed if your extraction
+                        was via global-workflow and NOT copied to $TANKDIR.
+
+6.  Move the data and html files to the web server (emcrzdm).  If you're on wcoss2 and  
+have set up password free access to web server for your account, RadMon_IG_glb.sh will 
+queue the transfer script and move the files to the server.  Alternately the transfer 
+script, RunTransfer.sh, can be run from the command line.
+
```
Comment (ADCollard) [inline src/Radiance_Monitor/README:82]: Maybe add usage for Run_Transfer.sh?

File: src/Radiance_Monitor/README [hunk 4]
```diff
@@ -1,65 +1,75 @@
...
+1.  Run GSI-monitor/ush/build.sh.  This builds all necessary executables.
+
+2.  The GSI-monitor sets default values for necessary storage, work, and log file 
+locations in GSI-monitor/parm/Mon_config.  If you want to override the defaults
+the important settings are: 
+
+tankdir  -- the location for extracted data storage
+ptmp     -- log file location
+stmp     -- work space
+
```
Comment (ADCollard) [inline src/Radiance_Monitor/README:28]: Also WEBUSER?

What about MAIL_TO in `src/Radiance_Monitor/parm/RadMon_user_settings`?

**Review by ADCollard — CHANGES_REQUESTED** [review]
So I've suggested a small number of changes.   I have only gone through the RadMon process for now, but the other READMEs look OK (some of the RadMon changes apply to the other components too)

**Review by ADCollard — CHANGES_REQUESTED** [review]
So I've suggested a small number of changes.   I have only gone through the RadMon process for now, but the other READMEs look OK (some of the RadMon changes apply to the other components too)

File: README.md (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply README.md]: Done.

File: src/Radiance_Monitor/README (hunk 2 above)
Comment (EdwardSafford-NOAA) [reply src/Radiance_Monitor/README]: Done, and for other READMEs as well.

File: src/Radiance_Monitor/README (hunk 3 above)
Comment (EdwardSafford-NOAA) [reply src/Radiance_Monitor/README:82]: Done, and for others as well.

File: src/Radiance_Monitor/README (hunk 4 above)
Comment (EdwardSafford-NOAA) [reply src/Radiance_Monitor/README:28]: Done.

**Review by ADCollard — APPROVED** [review]

**Pull request merged**